lib/template.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
-const { join } = require('path');
+const { join, extname } = require('path');
 const { readFileSync } = require('fs');
 let sitemapTmpl;
 const { encodeURL } = require('hexo-util');
 
 module.exports = function(config) {
-  if (sitemapTmpl) return sitemapTmpl;
-
+  // if (sitemapTmpl) return sitemapTmpl;
```

Comment:
Would you please delete a comment out code?

Reply:
ok